src/components/Spinner/readme.md

```diff
@@ -98,25 +98,17 @@ const Icon = styled.div.attrs(props => {
 import React from 'react';
 import { Spinner } from 'react-rainbow-components';
 
-const containerStyles = {
-    backgroundColor: '#061c3f',
-    borderRadius: '0 0 0.875rem 0.875rem',
-};
-
 const textStyles = {
     color: '#fff',
 };
 
-<div
-    className="rainbow-background-color_white rainbow-align-content_center rainbow-position_relative rainbow-p-vertical_xx-large"
-    style={containerStyles}
->
+<InverseContainer className="rainbow-p-vertical_large rainbow-align-content_center rainbow-flex_wrap">
```

Comment:
when selecting dark theme spinner inverse component is not showing correctly
<img width="804" alt="Screen Shot 2020-02-22 at 2 11 29 PM" src="https://user-images.githubusercontent.com/24737097/75098559-3a8fd880-557d-11ea-80c9-620cf33494a7.png">